docs_src/pipelinebenchmarking.md

```diff
@@ -15,43 +18,46 @@ Benchmark scripts are containerized inside docker, depending on platforms/hardwa
     Build command may take a while to run depending on your internet connection and machine specifications.
 
 ## Step 2: Run Benchmark
-The benchmark.sh shell script is located under `benchmark_scripts` directory under the base directory.  Before executing this script,
+The `benchmark.sh` shell script is located under `benchmark_scripts` directory under the base directory.  Before executing this script,
 change the current directory to directory `benchmark_scripts`.
 
+This script will start benchmarking a specific number of pipelines or can start stream density benchmarking based on the desired FPS to reach.  
+Before one can run pipeline benchmarking for a given case, one has to determine several inputs listed as follows:
+
 ### Determine the input source type
 
-### RTSP
+The benchmark script can take one of different types from video input source as follows:
+
+### Real Time Streaming Protocol (RTSP)
 
     --inputsrc rtsp://127.0.0.1:8554/camera_0
 
-- **__NOTE:__** using RTSP source with the benchmark.sh will automatically run the camera simulator. The camera simulator will start an RTSP stream for each video file found in the sample-media folder.
+- **__NOTE:__** using RTSP source with the benchmark.sh will automatically run the camera simulator. The camera simulator will start an RTSP stream for each video file found in the `sample-media` folder.
```

Comment:
is this the Note format you decided to use? just want to make sure everything is consistent.

Reply:
that's a good question: i think there is not bold type on the other  pages, i can remove the bold type to make it consistent with other pages

Reply:
oh and also use !!!